config/mbed/CMakeLists.txt

```diff
@@ -419,4 +419,19 @@ if (CONFIG_CHIP_PW_RPC_LIGHTING_PROTO)
     )
 endif(CONFIG_CHIP_PW_RPC_LIGHTING_PROTO)
 
-endif(CONFIG_CHIP_PW_RPC)
\ No newline at end of file
+endif(CONFIG_CHIP_PW_RPC)
+if("capsense" IN_LIST MBED_TARGET_LABELS)
```

Comment:
Not sure what was missed in my review by Tymoteusz but the capsense library build is part of the application, not the Matter library.